dali/python/backend_impl.cc

```diff
@@ -585,6 +585,20 @@ std::unique_ptr<Tensor<Backend> > TensorListGetItemImpl(TensorList<Backend> &t,
   return ptr;
 }
 
+template <typename Backend>
+std::shared_ptr<TensorList<Backend>> TensorListFromListOfTensors(py::list &list_of_tensors,
+                                                                 string &layout) {
+  auto tl = std::make_shared<TensorList<Backend>>(list_of_tensors.size());
+  TensorVector<Backend> tv(list_of_tensors.size());
+  for (size_t i = 0; i < list_of_tensors.size(); ++i) {
+    auto &t = list_of_tensors[i].cast<Tensor<Backend>&>();
+    tv[i] = std::move(t);
+  }
+  tl->Copy(tv, 0);
```

Comment:
I think we probably should have a sync. We might do a Copy on some other stream (we have this stream pool that creates stream per-device), and sync on that stream, as syncing stream 0 might be a bit over the top.

Reply:
done